src/app/shared/services/layout.service.ts

```diff
@@ -8,6 +8,12 @@ export interface FullFrameContent {
   animated: boolean,
 }
 
+interface ConfigureOptions {
+  expanded: boolean,
```

Comment:
add some "doc" please. I am not sure a "layout expanded" makes sense out of context.

Edit: maybe "in context" as well ;-). Isn't there a better word for that? `fullFrameContent`? ... ?

Edit2: ah I now see the doc is in the function spec... is it really useful to define this type here? cannot everything be in the function definition?

Reply:
fixed

Reply:
My 2 first comment were ignored actually. :-/

Reply:
I thought they were by moving the declaration in the function. Fixed.
I did some renaming and switched from `FullFrameContent.expanded` to `FullFrameContent.active`, it help renaming the configure fumction parameter